server/src/main/java/io/spine/server/procman/model/ProcessManagerClass.java

```diff
@@ -75,35 +74,33 @@ private ProcessManagerClass(Class<P> cls) {
     ProcessManagerClass<P> asProcessManagerClass(Class<P> cls) {
         checkNotNull(cls);
         @SuppressWarnings("unchecked")
-        ProcessManagerClass<P> result = (ProcessManagerClass<P>)
+        var result = (ProcessManagerClass<P>)
                 get(cls, ProcessManagerClass.class, () -> new ProcessManagerClass<>(cls));
         return result;
     }
 
     @Override
     public ImmutableSet<CommandClass> commands() {
-        SetView<CommandClass> result =
-                union(super.commands(), commanderDelegate.commands());
+        var result = union(super.commands(), commanderDelegate.commands());
         return result.immutableCopy();
     }
 
     @Override
     public ImmutableSet<EventClass> events() {
-        SetView<EventClass> result =
-                union(reactorDelegate.events(), commanderDelegate.events());
+        var result = union(reactorDelegate.events(), commanderDelegate.events());
         return result.immutableCopy();
     }
 
     @Override
     public ImmutableSet<EventClass> domesticEvents() {
-        SetView<EventClass> result =
+        var result =
                 union(reactorDelegate.domesticEvents(), commanderDelegate.domesticEvents());
         return result.immutableCopy();
     }
 
     @Override
     public ImmutableSet<EventClass> externalEvents() {
-        SetView<EventClass> result =
+        var result =
                 union(reactorDelegate.externalEvents(), commanderDelegate.externalEvents());
```

Comment:
Please see if `union()` and `get()` calls in this file may be moved to the previous lines.

Reply:
Moved `union()`. But `get()` does not fit.